examples/http.rs

```diff
@@ -33,7 +33,7 @@ use jsonrpsee::types::jsonrpc::{JsonValue, Params};
 const SOCK_ADDR: &str = "127.0.0.1:9933";
 const SERVER_URI: &str = "http://localhost:9933";
 
-#[async_std::main]
+#[tokio::main]
```

Comment:
Just curious: why switch to Tokio here?

Reply:
`hyper` only works with `tokio`, Pierre spawned a background thread around that just to support both any runtime.

If we go ahead with this change it should be clearly documented or spawn a background thread for it again.